### PR TITLE
UCL-451 Update storybook from V7 to V8

### PR DESCRIPTION
**Pull request by MichalGorskiOnegini**

### CODE REVIEW

File: stories/Form/Elements/MultiSelect.stories.tsx [hunk 1]
```diff
@@ -21,8 +21,7 @@ import MultiSelectDocumentation from "./MultiSelect.mdx";
 import { MultiOption } from "../../../src/components/Form/Select/MultiSelect/MultiOption";
 import { MultiSelectProps } from "../../../src/components/Form/Select/Select.interfaces";
 import { conditionalPlay } from "../../../.storybook/conditionalPlay";
-import { expect } from "@storybook/jest";
-import { userEvent, waitFor, within } from "@storybook/testing-library";
+import { userEvent, waitFor, within, expect } from "@storybook/test";
```
Comment (szmejk) [inline stories/Form/Elements/MultiSelect.stories.tsx:24]: see above, over here this new utilty package is used

File: stories/Typography/Typography.stories.tsx [hunk 2]
```diff
@@ -29,6 +29,10 @@ const meta: Meta = {
   parameters: {
     docs: {
       page: TypographyDocumentation
+    },
+    chromatic: {
+      delay: 300,
+      diffThreshold: 0.9 //To not look at font change, checked with https://6262c53f521620003ac2ff49-ukmsdlppcb.chromatic.com/?path=/story/stories-diff-threshold-check--test-yours-out
```
Comment (szmejk) [inline stories/Typography/Typography.stories.tsx:35]: nice!

**Review by szmejk — CHANGES_REQUESTED** [review]

File: stories/Typography/Typography.stories.tsx (hunk 2 above)
Comment (MichalGorskiOnegini) [reply stories/Typography/Typography.stories.tsx:35]: kudos to @pawel-napieracz :D

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -215,7 +215,7 @@ const { component } = createComponent(defaultParams => ({ ...defaultParams, prop
 "Component Library MDX": {
   "prefix": "clmdx",
   "body": [
-    "import { Story, Canvas, Title, Subtitle, ArgsTable, PRIMARY_STORY } from '@storybook/addon-docs';",
```
[the remark this answers is not in the document]
Comment (MichalGorskiOnegini) [reply CONTRIBUTING.md:218]: good catch

File: src/components/Tiles/Tile.test.tsx [hunk 4]
```diff
@@ -22,7 +22,7 @@ import { ContextMenu } from "../ContextMenu/ContextMenu";
 import { IconButton } from "../Button/IconButton";
 import { ContextMenuItem } from "../ContextMenu/ContextMenuItem";
 import userEvent from "@testing-library/user-event";
-import { waitFor } from "@storybook/testing-library";
+import { waitFor } from "@testing-library/dom";
```
[the remark this answers is not in the document]
Comment (MichalGorskiOnegini) [reply src/components/Tiles/Tile.test.tsx:25]: it's not a storybook test :D

[the remark this answers is not in the document]
Comment (szmejk) [reply src/components/Tiles/Tile.test.tsx:25]: :see_no_evil:

**Review by szmejk — APPROVED** [review]

**Review by szmejk — APPROVED** [review]

**Pull request merged**